org.eclipse.xtext.ui.tests/src/org/eclipse/xtext/ui/tests/quickfix/ui/quickfix/QuickfixCrossrefTestLanguageQuickfixProvider.java

```diff
@@ -77,4 +80,20 @@ public void renameFixable(final Issue issue, IssueResolutionAcceptor acceptor) {
 			});
 		});
 	}
+	
+	@Fix(QuickfixCrossrefTestLanguageValidator.LOWERCASE)
+	public void fixLowercase(final Issue issue, IssueResolutionAcceptor acceptor) {
+		acceptor.accept(issue, "fix lowercase", "fix lowercase", null, new ITextualMultiModification() {
```

Comment:
A test that shows how to work with fixes that change the length of the document would be great.